cmd/compose/up.go

```diff
@@ -310,6 +315,7 @@ func runUp(
 			Wait:           upOptions.wait,
 			WaitTimeout:    timeout,
 			Watch:          upOptions.watch,
+			Prune:          upOptions.prune,
```

Comment:
I wonder if we should not pass directly [`WatchOptions`](https://github.com/docker/compose/blob/f70209cf15efbcc009531879cc4cf9c9082cfb81/pkg/api/api.go#L128)